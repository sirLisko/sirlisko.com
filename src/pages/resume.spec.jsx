import React from "react";
import { render } from "@testing-library/react";
import { StaticQuery } from "gatsby";

import Resume from "./resume.jsx";

beforeEach(() => {
  StaticQuery.mockImplementationOnce(({ render }) =>
    render({
      site: {
        siteMetadata: {
          title: `I am the title`,
        },
      },
    })
  );
});

describe("Resume Page", () => {
  it("should render properly", () => {
    const { asFragment } = render(<Resume />);
    expect(asFragment()).toMatchSnapshot();
  });
});
