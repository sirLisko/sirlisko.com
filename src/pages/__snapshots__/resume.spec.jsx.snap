// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Resume Page should render properly 1`] = `
<DocumentFragment>
  <div
    class="resume"
  >
    <section
      class="me"
    >
      <div
        class="me__face"
      >
        <a
          href="/"
        >
          <img
            alt="sirlisko face"
            src="https://sirlisko.com/images/sirlisko.svg"
          />
        </a>
      </div>
      <p
        class="me__intro"
      >
        <span>
          Luca Lischetti
        </span>
        <span>
          a.k.a. sirLisko
        </span>
        <span>
          Senior Software Engineer
        </span>
      </p>
    </section>
    <div
      class="wrapper"
    >
      <article
        class="skills"
      >
        <h1>
          Skills
        </h1>
        <div>
          <h2>
            main
          </h2>
          <ul>
            <li>
              main_1
            </li>
            <li>
              main_2
            </li>
            <li>
              main_3
            </li>
          </ul>
        </div>
        <div>
          <h2>
            misc.
          </h2>
          <ul>
            <li>
              misc_1
            </li>
            <li>
              misc_2
            </li>
            <li>
              misc_3
            </li>
          </ul>
        </div>
      </article>
      <article
        class="section experiences"
      >
        <h1>
          Experiences
        </h1>
        <div>
          <h2>
            dove? 
            <span>
              01/01
            </span>
          </h2>
          <p>
            I've done this this and this
          </p>
        </div>
        <div>
          <h2>
            donde? 
            <span>
              00/00 01/01
            </span>
          </h2>
          <p>
            I've done that that and that
          </p>
        </div>
      </article>
      <article
        class="section projects"
      >
        <h1>
          Side Projects
        </h1>
        <div>
          <h2>
            <a
              href="https://foo.bar/"
            >
              Foo
            </a>
          </h2>
          <p>
            foo
          </p>
          <p>
            bar
          </p>
          <p>
            FooBar :)
          </p>
        </div>
        <div>
          To see a full list of my side projects go to 
          <a
            href="/projects"
          >
            https://sirlisko.com/projects
          </a>
          .
        </div>
      </article>
      <article
        class="section education"
      >
        <h1>
          Education
        </h1>
        <p>
          Degree in Computer Science @ Università degli studi dell’Insubria & Universidad de Salamanca.
        </p>
        <p>
          Thesis on Assistive Technology - 
          <a
            data-beacon="true"
            href="http://assistivetechnology.it"
          >
            http://assistivetechnology.it
          </a>
           (sorry link in Italian).
        </p>
      </article>
      <article
        class="section extras"
      >
        <h1>
          Extras
        </h1>
        <p>
          I co-held a workshop about TDD in Javascript @ Università di Cesena.
        </p>
        <p>
          I’m a Volunteer and occasional collaborator of the Front-End Conference - 
          <a
            data-beacon="true"
            href="https://fromthefront.it"
          >
            From The Front
          </a>
          .
        </p>
        <p>
          I have a Cisco CCNA cetificate (Cisco Certified Network Associate).
        </p>
        <p>
          I have a training certificate as Fire Marshal.
        </p>
      </article>
      <article
        class="section interests"
      >
        <h1>
          Interests
        </h1>
        <p>
          I like very much 
          <a
            href="https://www.last.fm/user/sirlisko"
          >
            music
          </a>
          , 
          <a
            href="https://www.imdb.com/user/ur15844029/"
          >
            cinema
          </a>
          , 
          <a
            href="https://www.goodreads.com/sirLisko"
          >
            books
          </a>
          , and, last but not least, 
          <a
            href="https://pizzaclub.sirlisko.com/"
          >
            pizza
          </a>
          .
        </p>
        <p>
          I like Art, especially modern and 8-bit graphic (like old-school video games), travel, languages. I’m also an active member of 
          <a
            href="https://www.couchsurfing.com/people/sirlisko"
          >
            CouchSurfing community
          </a>
          .
        </p>
      </article>
      <article
        class="section contacts"
      >
        <h1>
          Contacts
        </h1>
        <p>
          My personal website 
          <a
            href="https://sirlisko.com"
          >
            https://sirlisko.com
          </a>
        </p>
        <p>
          My Github 
          <a
            href="https://github.com/sirlisko"
          >
            https://github.com/sirlisko
          </a>
        </p>
        <p>
          My LinkedIn 
          <a
            href="https://www.linkedin.com/in/lucalischetti/"
          >
            https://www.linkedin.com/in/lucalischetti/
          </a>
        </p>
        <p>
          Drop me an email at 
          <a
            href="mailto:luca@sirlisko.com"
          >
            luca@sirlisko.com
          </a>
          .
        </p>
        <p>
          A print-friendly version of this: 
          <a
            href="https://sirlisko.com/cv/alt.pdf"
          >
            https://sirlisko.com/cv/alt.pdf
          </a>
        </p>
      </article>
    </div>
  </div>
</DocumentFragment>
`;
